.env

```diff
@@ -87,7 +87,7 @@ export LOG_LEVEL="info"
 
 export KRATOS_MASTER_USER_PASSWORD="passwordHardtoFindWithNumber123"
 export KRATOS_PG_HOST="localhost"
-export KRATOS_PG_PORT="5433"
+export KRATOS_PG_PORT="5432"
```

Comment:
I dont see changes in docker compose.. are you sure about this?

Reply:
Oh right, this assumes we now target tilt deps and not docker-compose deps anymore. It's probably irrelevant since we're planning to find a way to remove .env and have all the things we would normally need for local dev come through buck either way.

We have some cleaning up of docs, deps and local workflows to do after this PR